constrain/library/G36OutdoorAirDamperPositionForReturnFanAirflowTracking.py

```diff
@@ -1,34 +1,63 @@
 """
-G36 2021
-
 ### Description
 
 Section 5.16.2.3
+- Supply air temperature shall be controlled to setpoint using a control loop whose output is mapped to sequence the heating coil (if applicable), outdoor air damper, return air damper, and cooling coil.
 
-### Verification Logic
+### Code requirement
 
-```
-if abs(oa_p - max_oa_p) < oa_p_tol
+- Code Name: ASHRAE Guideline 36
+- Code Year: 2021
+- Code Section: 5.16.2 Supply Air Temperature Control
+- Code Subsection: 5.16.2.3 Outdoor Air Damper Control with Return Fan Airflow Tracking
+
+### Verification Approach
+
+The verification checks that the outdoor air damper maintains its maximum position within an acceptable tolerance. This position ensures proper building pressurization when using return fan airflow tracking control.
```

Comment:
interpretation is wrong, wrt code language, may be an issue with logic

Reply:
When I saw the implementation logic, I believe the explanation is reasonable. Additionally, I was going to copy the description from the library file ("Verify that the outdoor air damper operation follows the Guideline 36 recommendations for supply air temperature setpoint operations"). However, I think the language in the current code makes more sense to me. 

Please let me know if you have different thoughts.